Makefile

```diff
@@ -1,75 +1,40 @@
-all: init test build
-build: init docs/index.yaml docs/es/all.yaml docs/es/all-latest.yaml
-
-CHART := enterprise-suite
-export VERSION := $(shell scripts/export-chart-version.sh $(CHART))
-export RELEASE := $(CHART)-$(VERSION)
-CHART_LATEST := $(CHART)-latest
-export RELEASE_LATEST := $(CHART_LATEST)-$(VERSION)
-
-define banner
-	$(info === $@)
-endef
-
-docs/es/all.yaml: docs/$(RELEASE).tgz
-	$(call banner)
-	helm --namespace=lightbend template $< > $@
-
-docs/es/all-latest.yaml: docs/$(RELEASE_LATEST).tgz
-	$(call banner)
-	helm --namespace=lightbend template $< > $@
-
-docs/index.yaml: docs/$(RELEASE).tgz docs/$(RELEASE_LATEST).tgz
-	$(call banner)
-	helm repo index docs --url https://lightbend.github.io/helm-charts
-
-docs/$(RELEASE).tgz: $(CHART)/* $(CHART)/*/*
-	$(call banner)
-	helm package $(CHART) -d docs
-
-docs/$(RELEASE_LATEST).tgz: $(CHART)/* $(CHART)/*/*
-	$(call banner)
-	rm -rf build/$(CHART_LATEST)
-	cp -r $(CHART) build/$(CHART_LATEST)
-	scripts/munge-to-latest.sh build/$(CHART_LATEST)
-	helm package build/$(CHART_LATEST) -d docs
-
-# duplicate method here to generate the index, to avoid pulling in RELEASE as a dependency
-latest: init test docs/es/all-latest.yaml docs/$(RELEASE_LATEST).tgz
+# Top-level Makefile for creating the index of helm charts.
+
+# ALL CHARTS MUST:
+# - Have a file layout as described at https://docs.helm.sh/developing_charts/
+# - Have a Makefile with a recipe for a 'package' target.  This target should create
+#   the chart tarball and push it up to the helm-charts/docs directory.
+# - Have a Makefile with a recipe for a 'test' target.  This target is run by Travis
+#   to test the release.  That could be as simple as (the non-testing) Makefile:
+#      test: ;
+# 
+# The 'common.mk' file can be used to implement these targets if desired.
+
+# Collection of charts to process.
+# These are subdirectories of helm-charts.  Add to the list as required.
+#
+# To package/test a subset, just define a value on the command line.
+#    e.g. "make package CHARTS=sample-project"
+CHARTS = enterprise-suite enterprise-suite-latest reactive-sandbox #sample-project
+
```

Comment:
I think it would be nice to have a top level `all` target here which packages everything. E.g. `all: package docs/index.yaml`. So make by default builds all the charts.